### PR TITLE
FINERACT-821 Enforces(1) RedundantModifier Checkstyle

### PR DESCRIPTION
**Pull request by thesmallstar**
There are another 175 files that need to be fixed, I will send another PR for that. I have verified from my side that only "redundant modifiers" as caught by checkstyle are removed, but I request a detailed review if possible.

### CODE REVIEW

**Review by ptuomola — APPROVED** [review]

**Comment by ptuomola** [comment]
LGTM

**Comment by ptuomola** [comment]
@thesmallstar Do you want me to wait until you have the second set ready and then merge both together, or should I go ahead and merge this now?

**Comment by thesmallstar** [comment]
@ptuomola please merge this, that way we can avoid merge conflicts.

**Pull request merged**
